packages/frontend/styles/globals.css

```diff
@@ -13,3 +13,12 @@ h4 {
 p {
   margin: 1rem 0;
 }
+
+/* temprory classes for bordering on ui */
+.bd-red {
+  border: 1px solid red;
+}
+
+.bd-blue {
+  border: 1px solid blue;
+}
```

Comment:
These are not being used.

Reply:
sry this is for debugging borders on ui i use this to quickly add borders to check layouts etc.  
Sometimes I  forget to remove it. 

can we keep it for now until we go fully production release ??
because i have to remove it every time i push. 
on final build we can remove it.  if you don't mind ? 
whatever you say

Reply:
I think we shouldn't be adding debug stuff into the repo.

You can use "git update-index --assume-unchanged" if you want to do this.

Reply:
sure. let  me try this git hack